lion/io/Xml_element.h

```diff
@@ -68,9 +68,16 @@ class Xml_element
     //! See if a child exists
     bool has_child(const std::string& name) const;
 
+    //! See if parent exists
+    bool has_parent() const {if(_e->Parent()->ToElement()){return true;} else{return false;}}
```

Comment:
Replace by:

```C++
bool has_parent() const {return (_e->Parent()->ToElement() ? true : false); }
```

it's the best way to write this kind of one-liners. Easy simple and concise